site/_docs/powered_by.md

```diff
@@ -118,3 +118,9 @@ It uses Calcite to transparently rewrite queries to use materialized views.
 is a SQL interface to a
 <a href="https://tinkerpop.apache.org/">Apache TinkerPop</a>-enabled
 graph database.
+
+### HerdDB
```

Comment:
Thanks a lot for adding HerdDB to the powered_by page. Can you order the entry alphabetically? I can help merge the commit.

Reply:
@eolivelli Also please add a period at the end of the sentence.

@suez1224 Let me know if you need help publishing the web site. If you do, be careful not to deploy doc changes for features added since 1.16.

Reply:
@julianhyde actually, I dont know how to publish the web site. Is there any instruction?